tests/test-cases/Group22-Docker-Apps/22-10-logstash.robot

```diff
@@ -33,17 +33,17 @@ Logstash with stdin and stdout mapped
     Wait Until Keyword Succeeds  10x  6s  Check logstash logs  log1  Successfully started Logstash API endpoint
 
 Logstash with mapped volume log file
-    ${status}=  Get State Of Github Issue  6386
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 22-10-logstash.robot needs to be updated now that Issue #6384 has been resolved
-    #${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=vol1
-    #Should Be Equal As Integers  ${rc}  0
-    #${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --rm -v vol1:/mydata ${busybox} sh -c "echo 'Initial log message' > /mydata/my.log"
-    #Should Be Equal As Integers  ${rc}  0
-    #${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name log2 -dit -v vol1:/logs logstash -e 'input { file { path => "/logs/my.log" start_position => "beginning" } } output { stdout { } }'
-    #Log  ${output}
-    #Should Be Equal As Integers  ${rc}  0
-    #Wait Until Keyword Succeeds  10x  6s  Check logstash logs  log2  Initial log message
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=vol1
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --rm -v vol1:/mydata ${busybox} sh -c "echo 'Initial log message' > /mydata/my.log"
+    Should Be Equal As Integers  ${rc}  0
+
+    ${log_container}=  Set Variable  log2
```

Comment:
really not any benefit to this.. quite a few more keystrokes and doesn't create any clarity from my perspective...

Reply:
Ah that was in the previous test that I copied as a skeleton. I will probably remove it now that I am trying to get this thing merged.